parlai/tasks/wrapper/agents.py

```diff
@@ -144,3 +105,58 @@ def shutdown(self):
 
     def update_counters(self):
         self.task.update_counters()
+
+
+class LabelToTextTeacher(AbstractWrapperTeacher):
+    """
+    Teacher that will shift message['labels'][0] into message['text'] for whatever task
+    is specified with --label-to-text-task.
+
+    Because the dialogue history is effectively overwritten by this action, all episodes
+    will be flattened into one example each.
+    """
+
+    @classmethod
+    def add_cmdline_args(cls, argparser):
+        # Teacher has no args
+        parser = argparser.add_argument_group('LabelToText args')
+        parser.add_argument(
+            '-l2tt',
+            '--label-to-text-task',
```

Comment:
maybe we can change the argument to `--wrapper-task`  and then `__init__` could be implemented in the abstract teacher. Just an idea.

Reply:
I had that exact name before, lol. But yes, changing it back and implementing `__init__` :) thanks